### PR TITLE
[BREAKING] memo string->object

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: lib/persistent-knex-model-mixin.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = function PersistentKnexModelMixin (Model, knex, inOptions, user
     return this._applyFilter(this.constructor.convertToPersistent, this.getData())
   }
 
-  Model.prototype.save = function * (options) {
+  Model.prototype.save = function (options) {
```
Comment (MatthewPhinney) [inline lib/persistent-knex-model-mixin.js]: This was an unnecessary generator? This change doesn't affect any of the components, right?

Comment (sentientwaffle) [inline lib/persistent-knex-model-mixin.js]: Yes, and correct. The schema change is the breaking one.

File: schemas/FundsTemplate.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     },
     "memo": {
       "description": "Additional information related to the credit",
-      "type": "string"
+      "type": "object"
```
Comment (lumberj) [inline schemas/FundsTemplate.json]: Is this only used for transfer objects at the moment?

**Comment by emschwartz** [comment]
LGTM

**Comment by emschwartz** [comment]
LGTM (on the additions)

**Comment by lumberj** [comment]
LGTM

**Pull request merged**